topics/dev/tutorials/interactive-tools/tutorial.md

```diff
@@ -856,6 +813,10 @@ We have demonstrated how to pass an input file to the Docker container. But what
 >
 {: .tip}
 
+## Self-destruct script
+Web server applications will tend to keep running after the user has terminated the tool in Galaxy.
```

Comment:
> Web server applications will tend to keep running after the user has terminated the tool in Galaxy.

According to Helena's comment, shouldn't we remove this sentence ?

Reply:
Good point - have reworded this so it's not misleading.